playbooks/powervs-services.yml

```diff
@@ -14,7 +14,10 @@
 # limitations under the License.
 # ------------------------------------------------------------------------
 ---
-- hosts: all
+- name: Execute Ansible roles for PowerVS services and enable client services
```

Comment:
I think this file must be reverted. As roles will not be installed on os and will not be found. But run a test and check @AnaBil-k